machinelearning/pytorch.md

```diff
@@ -191,8 +189,8 @@ sudo apt install -y scons cmake
 cd $HOME
 git clone https://github.com/ARM-software/ComputeLibrary.git
 cd ComputeLibrary
-git checkout v22.11
-scons Werror=1 -j8 debug=0 neon=1 opencl=0 os=linux openmp=1 cppthreads=0 arch=armv8.2-a multi_isa=1 build=native
+git checkout v23.05.1
+scons Werror=1 -j8 debug=0 neon=1 opencl=0 os=linux openmp=1 cppthreads=0 arch=armv8a multi_isa=1 build=native
```

Comment:
I don't know the context here, but do we want to go backward to `armv8-a` instead of `armv8.2-a`?

Reply:
This is the minimum version, "multi_isa=1" takes care of bringing newer ISA features at runtime.